apps/api/src/app/layouts/usecases/update-layout/update-layout.use-case.ts

```diff
@@ -26,6 +26,10 @@ export class UpdateLayoutUseCase {
     });
     const databaseEntity = await this.getLayoutUseCase.execute(getLayoutCommand);
 
+    if (!command.isDefault && databaseEntity.isDefault) {
+      throw new ConflictException(`One default layout is required`);
+    }
```

Comment:
Not sure if this is the right thing to do. Here you are checking that the existing layout is a default. That doesn't mean that other layout is the default for the environment.

Reply:
This checks if the update (the data from the command) is to change default to false, and that previously (from the db) it was set to true. 
There could only be one default, so if it is changed from true to false, there is no other default. 
Or I might not understand what you mean :)

Reply:
Got it. I got it wrong and this is totally right. 👏🏻